src/modules/launcher/Wox.Infrastructure/Storage/StorageHelper.cs

```diff
@@ -0,0 +1,103 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.IO;
+using System.Windows.Markup;
+
+namespace Wox.Infrastructure.Storage
+{
+    public class StorageHelper
+    {
+        // This detail is accessed by the storage items and is used to decide if the cache must be deleted or not
+        public bool clearCache = false;
+
+        
+        private String currentPowerToysVersion = String.Empty;
+        private String FilePath = String.Empty;
+
+        // As of now this information is not pertinent but may be in the future
+        // There may be cases when we want to delete only the .cache files and not the .json storage files
+        private enum StorageType
+        {
+            BINARY_STORAGE = 0,
+            JSON_STORAGE = 1
+        }
+
+        // To compare the version numbers
+        public static bool Lessthan(string version1, string version2)
```

Comment:
Generally its a good idea to validate parameter input on public interfaces.   version1 and version2 shouldn't be null.